docs/docs/troubleshooting/TROUBLESHOOTING.md

```diff
@@ -2,15 +2,15 @@
 
 This guide contains general tips on how to investigate an application deployment that doesn't work correctly.
 
-??? tip "How do I uninstall an environment using a different Terraform configuration file?" 
-  
```

Comment:
There are lot's of unnecessary changes comes from your editor settings which makes the PR very noisy, and because all other have different setting, probably if anyone else makes another change in the files they will repeat this noise again. I suggest change the rules on your editor to avoid those changes. I see yours end trims all spaces which means change in many lines.

Reply:
Fixed by editing in GitHub. Somehow Atom would trim those spaces